site/static/compare.html

```diff
@@ -46,18 +47,20 @@ <h3>Cache states</h3>
         </ul>
     </div>
     <div id="content" style="display: none"></div>
-    <div id="settings">
-        <div id="commits" class="settings" style="text-align:left;">
-            <h3>Commits</h3>
-            Commit/Date A: <input width="100em" placeholder="YYYY-MM-DD or SHA" id="start-bound" /><br>
-            Commit/Date B: <input width="100em" placeholder="YYYY-MM-DD or SHA" id="end-bound" /><br>
-            <select id='stats' name="stat">
-            </select>
-            <div class="submit">
-                <a href="#" onClick="submit_settings(); return false;">Submit</a>
-            </div>
-        </div>
-    </div>
+    <form id="settings" action="">
+        <fieldset id="commits">
+            <legend>Commits</legend>
+            <label for="start-bound">Commit/Date A:</label>
+            <input width="100em" placeholder="YYYY-MM-DD or SHA" id="start-bound" /><br>
+            <label for="end-bound">Commit/Date B:</label>
+            <input width="100em" placeholder="YYYY-MM-DD or SHA" id="end-bound" /><br>
+        </fieldset>
+        <label for="stats">Choose a comparison method:</label>
+        <select id='stats' name="stat">
+        </select><br>
+        <input type="submit" value="Submit" onclick="submit_settings(); return false;">
```

Comment:
I'd include the buttons in the fieldset so the box is drawn around all of it.

Reply:
I don't have strong preference. To be honest, I'm not a web dev. This PR is result of googling around
plus some web basic knowledge. I'd leave it to maintainer to decide.